ReturnReminderManager/main.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jan 14 18:48:12 2021
+
+@author: Simon Kern
+"""
+
+import sys; sys.path.append('.');sys.path.append('..')
+import re
+import logging
+import time
+from datetime import datetime
+from leihlokal import LeihLokal
+from mail_client import MailClient
+from collections import defaultdict
+
+logging.basicConfig(level=logging.DEBUG)
+logging.getLogger().setLevel(logging.DEBUG)
+
+
+def get_reminder_template(customer, rentals):
+    several = len(rentals)>1
+    items = [rental.item for rental in rentals]
+    items_str = ", ".join([f"{item.name} (#{item.id})" for item in items])
+    s = f"""\
+Liebe/r {customer.firstname} {customer.lastname},
+
+wir hoffen, Sie hatten viel Freude an {'den' if several else 'dem'} bei uns geliehenen \
+{'Gegenständen' if several else 'Gegenstand'}!
+
+Heute wird die Rückgabe {'der' if several else 'des'} \
+{'Gegenstände' if several else 'Gegenstands'}: "{items_str}" fällig. \
+Wir bitten Sie, {'die Gegenstände' if several else 'den Gegenstand'} heute zu \
+den Öffnungszeiten zurück zu bringen.
+
+Falls die maximale Leihdauer von drei Wochen noch nicht erreicht ist, \
+können Sie ggf. die Ausleihe noch einmal per Telefon oder Mail verlängern. \
+Sollten Sie den Gegenstand jedoch nicht mehr benötigen, freuen sich die nachfolgenden \
+AusleiherInnen über eine zeitige Rückgabe.
+
+Sollte es sich heute um keinen Öffnungstag handeln, geben Sie {'die Gegenstände' if several else 'den Gegenstand'}\
+einfach am darauffolgenden Öffnungstag ab. Weitere Informationen finden Sie unter https://bitly.com/leihlokal
+
+Liebe Grüße, 
+das leih.lokal-Team
+
+Gerwigstr. 41, 76185 Karlsruhe
+Öffnungszeiten: Mo, Do: 15-19, Sa: 11-16
+Telefon: 0721/ 4700 4551
+http://www.buergerstiftung-karlsruhe.de/leihlokal/
+
+//
+
+Das leih.lokal ist eine ehrenamtliches Projekt von der Bürgerstiftung Karlsruhe.
+Wir arbeiten komplett spendenfinanziert und freuen uns daher über Ihre Spende.
+Mach doch auch mit und hilf uns die Welt ein bisschen nachhaltiger zu gestalten!
+
+// Diese Email wurde automatisch generiert. Sie kann daher Fehler enthalten. \
+Wir bitten dies zu entschuldigen.
+
+"""
+    return s
+
+def valid_email(email):
+  if not isinstance(email, str): return False
+  return bool(re.search(r"^[\w\.\+\-]+\@[\w]+\.[a-z]{2,3}$", email))
+
+
+if __name__ == '__main__':
+
+    leihlokal = LeihLokal()
+    mail_client = MailClient()
+
+    # not sure how well this plays for the github timezone?
+    today = datetime.now().date()
+    rentals_due_today = leihlokal.filter_rentals(lambda x: x.to_return_on==today)
```

Comment:
Did you test if this works? I think the timestamps in the database always are at midnight.

Reply:
Okay, so if they are at midnight and the script runs at 7am (github time?), this should work?

I'll test it in master and see if it sends them at the right time.